Azure-Samples/chat-with-your-data-solution-accelerator
# Add a azd CI pipeline

## Purpose
Use the azd cli to provision and deploy the accelerator
Fixes #117

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Check that we can successfully build and deploy the accelerator

![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/6254153/606155ee-89a6-4f05-aa19-c9c0e744eecd)
